_examples/federation/reviews/graph/schema.resolvers.go

```diff
@@ -11,23 +11,62 @@ import (
 	"github.com/99designs/gqlgen/_examples/federation/reviews/graph/model"
 )
 
+// ManufacturerID is the resolver for the manufacturerID field.
+func (r *productResolver) ManufacturerID(ctx context.Context, obj *model.Product, federationRequires map[string]interface{}) (*string, error) {
```

Comment:
New resolver function automatically generated and includes the `@requires` fields in the `federationRequires` map.